solr/solrj/src/java/org/apache/solr/common/params/CollectionParams.java

```diff
@@ -111,8 +111,8 @@ enum CollectionAction {
     DISTRIBUTEDAPIPROCESSING(false, LockLevel.NONE),
     LIST(false, LockLevel.NONE),
     CLUSTERSTATUS(false, LockLevel.NONE),
-    ADDREPLICAPROP(true, LockLevel.REPLICA),
-    DELETEREPLICAPROP(true, LockLevel.REPLICA),
+    ADDREPLICAPROP(true, LockLevel.NONE), // nocommit discuss
+    DELETEREPLICAPROP(true, LockLevel.NONE), // nocommit discuss
```

Comment:
@HoustonPutman I wonder what your opinion is here.  We've got one test `org.apache.solr.cloud.TestPullReplica#testSkipLeaderRecoveryProperty` that purposefully adds a replica that isn't happy and then sets a property (SKIP_LEADER_RECOVERY_PROP) that leads to it being happy.  I removed the lock level, which made the test pass so that it doesn't block on addReplica's happiness (final state).  That's not my reasoning to remove the lock level... I'm wondering it it's harmless to let replica properties be set on any replica at any time?

Maybe ideally the waitForFinalState should happen outside the LockLevel but that would be tricky as it amounts to a design change to command processing, giving commands a final stage outside the lock.